Doc/whatsnew/3.14.rst

```diff
@@ -2907,7 +2867,50 @@ New features
 
   (Contributed by Victor Stinner in :gh:`107954`.)
 
-* Add a new import and export API for Python :class:`int` objects (:pep:`757`):
+* Add :c:func:`Py_fopen` function to open a file.
+  This works similarly to the standard C :c:func:`!fopen` function,
+  accepting a Python object for the *path* parameter
```

Comment:
I suggest:

```suggestion
  but accepting a Python object for the *path* parameter
```

But I'm not a native English speaker.

Reply:
I don't think 'but' is needed here because we say "works similarly to" and then explain the differences in these lines here.